.npmrc

```diff
@@ -1 +1 @@
-legacy-peer-deps=true
+legacy-peer-deps=true
```

Comment:
Last nit, I don't think we actually need this turned on. I just blew away `package-lock.json` and the `node_modules` directory, did a fresh install and saw no warnings

Reply:
Cool, I'll remove it.